addons/mrp/models/stock_move.py

```diff
@@ -138,6 +138,14 @@ def action_assign(self, no_prepare=False):
         self.check_move_lots()
         return res
 
+    def _propagate_cancel(self):
```

Comment:
I'm not convinced this is necessary. I understand the logic of not canceling the raw materials when the PO/picking is canceled, but is it always what we want? If your product is MTO, maybe we really want to propagate the cancellation to all the moves of the chain.

To me, that's a change in behavior, and I wouldn't do it in v10 unless really necessary. OK for v11 though (if necessary, I haven't tried). Moreover, this is out of scope the issue reported.